### DISCUSSION TITLE
Upcoming release notes - 5.1.0

### DISCUSSION THREAD

**Discussion by itsvick** [Sunbird-inQuiry/Community#28] [General Discussions]
We have kickstarted Release-5.1.0. R 5.1 comprises 5 week release plan, where sprint 1 holds 2 weeks and Sprint 2 starts from the 3rd week and ends by the 5th week. Due to a delay in the previous release, we kept sprint 1 for 2 weeks.

Please checkout the plan mentioned below:

|               | Start Date  | End Date |
|------------|:-------------:|:---------------|
| Sprint 1 |  08 Aug 2022 | 19 Aug 2022 |
| Sprint 2 |    22 Aug 2022  | 09 Sep 2022 |
| PPV      | 12 Sep 2022 | 23 Sep 2022 |
| Prod | 26 Sep 2022 | |

Folks - the next version of inQuiry is expected to go live on 26th Sept. 2022. Here is where you can find out what's coming up as part of this release - https://inquiry.sunbird.org/use/release-notes/inquiry-release-v5.1.0-future

Do engage with us, if you have any questions.
@surendrasinghs @pallakartheekreddy @chitranshu-keshav @krgauraw @Nadiya496 @rajnishdargan

Cheers,
inQuiry BB team

**Comment by surendrasinghs** [comment]
@tarang27 Please share 5.1 plan soon

**Reply by pallakartheekreddy** [reply]
We have descoped the below items as the developer is busy with CSP changes. We will take the below items in the next release-5.2.0
1. inQuiry: Sonar & Circle Ci integration for pipeline
2. inQuiry: Question-set publish Flink job changes
3. inQuiry: Configuration Changes for Dependent Knowlg BB Components

**Reply by tarang27** [reply]
FYI @Prerna789 @sanyapawah
